### ISSUE TITLE
Can't Train Azure Model?

### ISSUE THREAD

**Issue by Philip2809** [dotnet/machinelearning-modelbuilder#1824]
Hello There!

I wanted to do a quick project with machine learning and followed:
https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/object-detection-model-builder

BUT I ran into a problem; Visual studio thinks that the training failed when it did not according to Microsoft Azure Machine Learning Studio. I did it 3 times but got the same resultat. I can assure you I have a good internet connection because I am writing this and talking to people on discord. I even restarted visual studio, nothing helped.

Any help appreciated,
Best regards
Philip Magyar

IMAGES:
![image](https://user-images.githubusercontent.com/51290462/136673047-0b6c2b59-f25e-41cf-b770-5dc0b31690f3.png)
![image](https://user-images.githubusercontent.com/51290462/136673048-8719cc3f-1f0a-4f1a-b591-ca0f07a8c0c8.png)

LOG:
[logfile.txt](https://github.com/dotnet/machinelearning-modelbuilder/files/7316338/logfile.txt)

**Comment by Philip2809** [comment]
BTW I downloaded the latest verion from:
https://marketplace.visualstudio.com/items?itemName=MLNET.07 earlier today, as I found; https://github.com/dotnet/machinelearning-modelbuilder/issues/1793

**Comment by Philip2809** [comment]
I think I found out what I did wrong but correct my if I am incorrect, I had WAY too little to train on, only 9 images and all failed or "skipped"?

**Comment by zewditu** [comment]
@ Philip2809 please send your data.
Assume your data is not in the proper format, are you expecting to get information on the data page if there is something wrong with your data instead of your training failure?

**Comment by zewditu** [comment]
@JakeRadMSFT  @LittleLittleCloud  and @beccamc  arr all data issues in OD notified to the user in data page?

**Comment by beccamc** [comment]
@v-Hailishi Can you try to reproduce this with the training data above?

**Comment by v-Hailishi** [comment]
@beccamc This issue is not repro on the latest main branch: 16.13.5.2222101 when I using above training data [ml_train_data.zip](https://github.com/dotnet/machinelearning-modelbuilder/files/7340646/ml_train_data.zip)
![image](https://user-images.githubusercontent.com/99375895/165501098-ccc9d3a6-ebe0-4ac5-adf3-b52286f86fbd.png)
Run link in Azure portal: 
https://ml.azure.com/experiments/id/8ad34cd8-0b2e-445b-a3a7-5760c3306ece/runs/AutoML_5dd26aea-c5d5-4d2d-902a-ea6d8ebdb336?wsid=/subscriptions/83635e26-687a-4333-a2a0-33e3cd1f573a/resourceGroups/FullTestPass/providers/Microsoft.MachineLearningServices/workspaces/FullTestPass&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#details

**Comment by beccamc** [comment]
@Philip2809 Can you update your Model Builder version and see if the problem is still happening?

**Comment by Philip2809** [comment]
Hello! I no longer use this to image recognition, I used Custom Vision for a while to make a model and use it but later found out it wasn't what I was looking for. Also it turned out image recognition was not really used in the project I was working on so I stopped using it. 

Sadly I no longer have the project where I was testing this stuff out as I have a new computer, the old one was erased. It it trains as it should now I would guess either the bug was fixed or something was wrong in my environment. 

Best Regards
Philip Magyar
